lib-injection/sitecustomize.py

```diff
@@ -77,6 +77,10 @@ def _inject():
             _log("failed to load ddtrace module: %s" % e, level="error")
             return
         else:
+            # Similar to SSI, this environment variable allows downstream components to know that the library was
```

Comment:
@Kyle-Verhoog, any bad smells from this?  I tried to align things with how SSI works at the top level, but I guess we'll clobber whatever value is here (in particular:  if the environment variables changes from a truthy-ish value to some kind of enum or whatever).

Reply:
Makes total sense to me.  I made two related changes in 4ef8cbf:

* Replace the `DD_LIBRARY_INJECTION` with a direct setting of `DD_PROFILER_EXPORTER_LIBDD_ENABLE` (or whatever that crazy environment variable actually is)
* Remove the injection configuration and any references to it

I think we're good here, then!